lib/Zonemaster/CLI.pm

```diff
@@ -1,7 +1,8 @@
 # Brief help module to define the exception we use for early exits.
 package Zonemaster::Engine::Exception::NormalExit;
-use Moose;
-extends 'Zonemaster::Engine::Exception';
+use 5.014002;
+use warnings;
+our @ISA = qw( Zonemaster::Engine::Exception );
```

Comment:
Why not `use parent Zonemaster::Engine::Exception`?

Reply:
Fixing.